homeassistant/components/lastfm/entity.py

```diff
@@ -0,0 +1,23 @@
+"""LastFM Entity."""
```

Comment:
I don't think you need to create the base entity class for now.
`device_info` is not allowed until config entry is implemented, so this whole module can be removed for now.